New Featured Artist: BilliumMoto (#2255)
ppy/osu-wiki: news/2019-05-15-new-featured-artist-billiummoto.md

---
layout: post
title: "New Featured Artist: BilliumMoto"
date: 2019-05-15 11:00:00 +0000
---

The winner of the first ever osu!remix contest enters the scene as osu!'s latest Featured Artist! Say hello to [**BilliumMoto**](https://osu.ppy.sh/beatmaps/artists/38)!

> [![](https://assets.ppy.sh/artists/38/header.jpg)](https://osu.ppy.sh/beatmaps/artists/38)

Reviewer: ```suggestion
![](https://assets.ppy.sh/artists/38/header.jpg)
```
header links don't do anything now

We're delighted to host **8** unique tracks from him, all of which are now available over at [his Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/38).

Here's a quick sample of a few:

### Dazzling (Short Ver.)

<audio controls>
    <source src="https://assets.ppy.sh/artists/38/previews/930.mp3" type="audio/mpeg">
</audio>

### HDHR (osu!remix 2016 winner)

<audio controls>
    <source src="https://assets.ppy.sh/artists/38/previews/932.mp3" type="audio/mpeg">
</audio>

### four veiled stars

<audio controls>
    <source src="https://assets.ppy.sh/artists/38/previews/934.mp3" type="audio/mpeg">
</audio>

### life flashes before weeb eyes

<audio controls>
    <source src="https://assets.ppy.sh/artists/38/previews/935.mp3" type="audio/mpeg">
</audio>

### Gimme More

<audio controls>
    <source src="https://assets.ppy.sh/artists/38/previews/931.mp3" type="audio/mpeg">
</audio>

--------

> As always, all of the tracks above plus the rest available on [**BilliumMoto**'s' Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/38) are 100% free for use in modding and mapping, and come packaged in pre-timed game files. Handy!

Reviewer: ```suggestion
As always, all of the tracks above plus the rest available on [**BilliumMoto**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/38) are 100% free for use in modding and mapping, and come packaged in pre-timed beatmaps. Handy!
```

> We're unofficially declaring May as the "home-grown" month for the Featured Artist project, and boy do we have a surprise slated for our next release at the end of the month. Keep your eyes and ears peeled, you won't want to miss out!

Reviewer: ```suggestion
We're unofficially declaring May as the "home-grown" month for the Featured Artist project, and boy do we have a surprise slated for our next release at the end of the month. Keep your eyes and ears peeled; you won't want to miss out!
```
